src/pages/mypage/components/missioncard/MissionList.tsx

```
@@ -0,0 +1,141 @@
+    return copied;
+  }, [baseList, timeSort, category]);
+
+  useEffect(() => {
+    setVisibleCount(loadSize);
+  }, [list.length, loadSize]);
+
+  const visibleList = list.slice(0, visibleCount);
+  const hasMore = visibleCount < list.length;
+
+  useEffect(() => {
+    if (!hasMore) return;
```

Biome (2.3.13)

[error] This hook specifies more dependencies than necessary: loadSize.

This dependency can be removed from the list.

React relies on hook dependencies to determine when to re-compute Effects.
Specifying more dependencies than required can lead to unnecessary re-rendering
and degraded performance.
Unsafe fix: Remove the extra dependencies from the list.


(lint/correctness/useExhaustiveDependencies)